source/configuration/read.txt

```diff
@@ -181,10 +169,10 @@ You can configure the following properties to read from MongoDB:
 Partitioner Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. _conf-mongosamplepartitioner:
+.. _conf-samplepartitioner:
```

Comment:
I didn't check whether the old ref was linked to by anything, or whether there might be a search engine pointing to this section, but it's safest to keep the old ref as well as create the new one.

E.g.
```
.. _conf-mongosamplepartition:
.. _conf-samplepartitioner:
```